### PR TITLE
Implement FileHandle.nullDevice

### PR DESCRIPTION
**Pull request by kareman**
Not sure if the unit test is needed. I may be testing OS functionality instead of Foundation.

### CODE REVIEW

File: Foundation/NSFileHandle.swift [hunk 1]
```diff
@@ -214,11 +214,15 @@ extension FileHandle {
     open class var standardError: FileHandle {
         return _stderrFileHandle
     }
-    
+
+    internal static var _nulldeviceFileHandle: FileHandle = {
+        return FileHandle(forUpdatingAtPath: "/dev/null")!
```
Comment (parkera) [inline Foundation/NSFileHandle.swift]: What's the portability story around `/dev/null`? 

For what it's worth, the Darwin Foundation implementation here is a simple subclass of FileHandle that returns empty `Data` and values of `0` (except for file handle, which is `-1`).

**Comment by kareman** [comment]
@parkera this should be more like the Darwin implementation.

**Comment by kareman** [comment]
Any news on this?

**Comment by parkera** [comment]
Oh, sorry! This looks good to me.

**Comment by parkera** [comment]
@swift-ci please test and merge

**Pull request merged**